### PR TITLE
Polaris PSA certified connector

### PR DESCRIPTION
**Pull request by repl-anand-vikram**
---
When submitting a connector, please check the following conditions for your PR to ensure a smooth certification process.

- [x] I attest that the connector works and I verified by deploying and testing all the operations. 
- [x] I attest that I have added detailed descriptions for all operations and parameters in the swagger file.
- [x] I validated the swagger file, `apiDefinition.swagger.json`, by running `paconn validate` command.
- [x] If this is a certified connector, I confirm that `apiProperties.json` has a valid brand color and doesn't use an invalid brand color, `#007ee5` or `#ffffff`. If this is an independent publisher connector, I confirm that I am not submitting a connector icon.

If you are an Independent Publisher, you must also attest to the following to ensure a smooth publishing process:
- [ ] I have named this PR after the pattern of "Connector Name (Independent Publisher)" ex: HubSpot Marketing (Independent Publisher)
- [ ] Within this PR markdown file, I have pasted screenshots that show: 3 unique operations (actions/triggers) working within a Flow. This can be in one flow or part of multiple flows. For each one of those flows, I have pasted in screenshots of the Flow succeeding. 
- [ ] Within this PR markdown file, I have pasted in a screenshot from the Test operations section within the Custom Connector UI.
- [ ] If the connector uses OAuth, I have provided detailed steps on how to create an app in the readme.md.

### CODE REVIEW

File: certified-connectors/Polaris PSA/apiDefinition.swagger.json
Comment (sriyen-msft) [inline certified-connectors/Polaris PSA/apiDefinition.swagger.json:917]: @repl-anand-vikram It looks like an operation Id rather than a summary value, please provide a valid summary value.

File: certified-connectors/Polaris PSA/apiDefinition.swagger.json
Comment (sriyen-msft) [inline certified-connectors/Polaris PSA/apiDefinition.swagger.json:943]: @repl-anand-vikram If it doesn't need, please remove it - Applies any property in the swagger.

File: certified-connectors/Polaris PSA/apiDefinition.swagger.json
Comment (sriyen-msft) [inline certified-connectors/Polaris PSA/apiDefinition.swagger.json:965]: @repl-anand-vikram The description values of properties should be free from grammatical and spelling errors. Please update them with valid description values. 

Applies this comment for all other similar values.

File: certified-connectors/Polaris PSA/apiProperties.json
Comment (sriyen-msft) [inline certified-connectors/Polaris PSA/apiProperties.json:43]: @repl-anand-vikram White background color is not allowed, please update it to some other color.

Documentation - [Design an icon for your connector](https://docs.microsoft.com/en-us/connectors/custom-connectors/certification-submission#design-an-icon-for-your-connector)

**Review by sriyen-msft — CHANGES_REQUESTED** [review]

**Review by sriyen-msft — CHANGES_REQUESTED** [review]
@repl-anand-vikram You need to sign CLA in the PR to get it approved and merged

File: certified-connectors/Polaris PSA/apiDefinition.swagger.json
Comment (sriyen-msft) [inline certified-connectors/Polaris PSA/apiDefinition.swagger.json:558]: @repl-anand-vikram There are many description values in the swagger need to be fixed, please update them

**Review by sriyen-msft — CHANGES_REQUESTED** [review]
@repl-anand-vikram You need to sign CLA in the PR to get it approved and merged

File: certified-connectors/Polaris PSA/apiProperties.json
Comment (sriyen-msft) [reply certified-connectors/Polaris PSA/apiProperties.json:43]: @repl-anand-vikram, Can you please fix this to get this PR approved

**Comment by sriyen-msft** [comment]
@repl-anand-vikram You need to sign CLA in the PR to get it approved and merged

**Review by sriyen-msft — CHANGES_REQUESTED** [review]
@repl-anand-vikram You need to sign CLA in the PR to get it approved and merged

**Comment by repl-anand-vikram** [comment]
> @repl-anand-vikram You need to sign CLA in the PR to get it approved and merged

@sriyen-msft we are reviewing the CLA with our legal team. while we do not see any concerns but we are waiting for formal approval. We will probably sign it on Monday. Thanks for following up

**Comment by repl-anand-vikram** [comment]
> > @repl-anand-vikram You need to sign CLA in the PR to get it approved and merged
> 
> @sriyen-msft we are reviewing the CLA with our legal team. while we do not see any concerns but we are waiting for formal approval. We will probably sign it on Monday. Thanks for following up

@sriyen-msft I have signed the CLA. Please proceed with the review

**Review by sriyen-msft — APPROVED** [review]

**Pull request merged**
